### DISCUSSION TITLE
Bin/Console on docker

### DISCUSSION THREAD

**Discussion by varunpan** [kimai/kimai#3508] [Questions & Answers] [answered]
Hi, 

this is probably a really simple thing, but i cant figure it out. I cannot execute any functions using bin/console.
I am running Kimai in docker with MySQL

I keep getting the following error:

```
docker exec kimai bin/console kimai:import:timesheet --help
OCI runtime exec failed: exec failed: unable to start container process: exec: "bin/console": stat bin/console: no such file or directory: unknown
```
I think I missed a step somewhere but I'm not sure.

**Comment by kevinpapst** [comment, marked as answer]
The director is wrong, it is not `bin/console` - check the docs in the docker report at https://github.com/tobybatch/kimai2

**Reply by varunpan** [reply]
Thank you, I knew it was something silly, since i was moving too fast!
